isic_kredit/ViewControllers/CreditViewController.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  CreditViewController.swift
+//  isic_kredit
+//
+//  Created by Petr Dusek on 07/11/2018.
+//  Copyright © 2018 Petr Dusek. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import Alamofire
+import SwiftKeychainWrapper
+import SwiftSoup
+import JGProgressHUD
+
+class CreditViewController: UIViewController {
+    
+    weak var reloadButton: UIButton!
+    weak var accountButton: UIButton!
+    weak var moneyLabel: UILabel!
+    var hud = JGProgressHUD(style: .light)
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        accountButton.addTarget(self, action: #selector(accountButtonTapped(_:)), for: .touchUpInside)
+        reloadButton.addTarget(self, action: #selector(reloadButtonTapped(_:)), for: .touchUpInside)
+        
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        if KeychainWrapper.standard.string(forKey: "username")==nil && KeychainWrapper.standard.string(forKey: "password")==nil{
```

Comment:
Práci s keychainem by bylo vhodný wrapnout do nějaký další třídy stejně jako další logiku, ne to takhle střelit rovnou do VC. Pracuješ s tím na dalších 3 místech a všude kopíruješ string `"username"`, to je náchylný na chyby.

Reply:
Je hotova trida KeyChain ktera to resi